main/main.cpp

```diff
@@ -2144,9 +2144,10 @@ Error Main::setup(const char *execpath, int argc, char *argv[], bool p_second_ph
 	}
 
 	GLOBAL_DEF("display/window/ios/allow_high_refresh_rate", true);
-	GLOBAL_DEF("display/window/ios/hide_home_indicator", true);
-	GLOBAL_DEF("display/window/ios/hide_status_bar", true);
-	GLOBAL_DEF("display/window/ios/suppress_ui_gesture", true);
+	GLOBAL_DEF_BASIC("display/window/ios/hide_home_indicator", false);
```

Comment:
This default value is changed, this does affect compatibility, what's the reason for his change?

Reply:
Hiding the home bar indicator and suppressing it are mutually exclusive, it was either a misunderstanding or my inexperience that caused the defaults to be this way in the first place. This weirdness of mutually exclusive options being possible to enable is fixed by Option 2.